### PR TITLE
Add data analysis introduction

### PR DESCRIPTION
**Pull request by dschust-r**
I've added a brief introduction to data analysis and will edit this according to the types of experiments discussed in the MS.

### CODE REVIEW

**Review by jessegmeyerlab — CHANGES_REQUESTED** [review]
Thanks for contributing! This is a great start here. I forgot to put in the pull request template that each line should be one sentence to make tracking diffs easier. I tried adding a pull request with those suggestions and a few minor rewordings and additions

**Review by jessegmeyerlab — CHANGES_REQUESTED** [review]
Looks like I accidentally just made the suggested changes without asking for your approval - there is a way to do this let me try again. Thanks for your patience while I learn manubot and github

**Review by dschust-r — COMMENTED** [review]
Looks good to me. 
Seems like I can't approve your changes since I am the author of the pull-request (otherwise I would approve my own pull-request). Thank you for adding the changes right away 😃

I think a table with link to the website + primary citation would be a great idea!

**Comment by jessegmeyerlab** [comment]
@agitter how do I suggest changes to the lines in someone's pull request like I saw when I submitted to the covid review? Github help says there should be a +/- button but I don't see one?

**Comment by dschust-r** [comment]
You can open the commit and once you hover over a line you should see the +

**Comment by jessegmeyerlab** [comment]
also @dschust-r I merged your other PR with structure changes before this one then this one couldn't merge because the files were named different. There is also a way to merge that PR into this one. I had to roll back that PR to make this one compatible  will make sure those changes get made. @agitter do you know how to do this too?

**Comment by jessegmeyerlab** [comment]
> You can open the commit and once you hover over a line you should see the +

When I click that it lets me comment but I don't see how to suggest an edit?

**Comment by dschust-r** [comment]
> > You can open the commit and once you hover over a line you should see the +
> 
> When I click that it lets me comment but I don't see how to suggest an edit?

Ah sorry, I think that only works in the "Files changed" tab. Same way but in the Files Changed section click on this button:

image

File: content/11.raw-data-analysis.md
Comment (jessegmeyerlab) [inline content/11.raw-data-analysis.md:20]:
```suggestion
<!-- test -->
```

File: content/11.raw-data-analysis.md
Comment (jessegmeyerlab) [inline content/11.raw-data-analysis.md]:
```suggestion
- DIA-NN [@DOI:https://doi.org/10.1038/s41592-019-0638-x]
```

**Review by jessegmeyerlab — CHANGES_REQUESTED** [review]
okay thanks! lets test this with a minor change suggestion

**Comment by jessegmeyerlab** [comment]
@dschust-r can you add to this PR your info for authorship in metadata.yml in the content folder? just copy and edit the entry for me. Or let me know the relevant info and I can do it for you

**Comment by dschust-r** [comment]
> @dschust-r can you add to this PR your info for authorship in metadata.yml in the content folder? just copy and edit the entry for me. Or let me know the relevant info and I can do it for you

Will do! I can also make the other PR with the new section after this one has been merged, then it should work. 

The edit suggestions seem to work now!

**Comment by jessegmeyerlab** [comment]
Awesome thanks. I made a new PR with my other account with the new section just now. I think I probably have to merge it with this branch before merging to main

**Comment by jessegmeyerlab** [comment]
Awesome! first merged pull request!!! Thanks @dschust-r for contributing

**Comment by agitter** [comment]
> Ah sorry, I think that only works in the "Files changed" tab.

Yes, this is how I typically make suggestions using the button in the screenshot above.  Note that if you accumulate a lot of these suggestions, you can add all of them to a single batch and then make 1 commit with that batch of changes instead of accepting each suggestion as its own commit.

> @agitter do you know how to do this too?

I'd have to look at the branches to say for sure.  One workflow is to merge whichever pull request / branch is ready first to the main branch and then deal with any new conflicts subsequently in other open pull requests.  It is possible to merge small conflicts through a button in the pull request.  Large and messy conflicts are best handled locally.

It is also possible to merge pull request branch 1 into pull request branch 2 locally in git before merging either pull request.  Maybe that can minimize conflicts, but we haven't usually used that workflow.

Overall, it is helpful if contributors do their best to not edit the same sentences or paragraphs of the manuscript at the same time (in separate pull requests) because that will create more conflicts.  Opening a draft pull request even if it isn't ready for final review and merging yet is one way to help notify everyone which parts are actively being edited.

**Comment by dschust-r** [comment]
> > Ah sorry, I think that only works in the "Files changed" tab.
> 
> Yes, this is how I typically make suggestions using the button in the screenshot above. Note that if you accumulate a lot of these suggestions, you can add all of them to a single batch and then make 1 commit with that batch of changes instead of accepting each suggestion as its own commit.
> 
> > @agitter do you know how to do this too?
> 
> I'd have to look at the branches to say for sure. One workflow is to merge whichever pull request / branch is ready first to the main branch and then deal with any new conflicts subsequently in other open pull requests. It is possible to merge small conflicts through a button in the pull request. Large and messy conflicts are best handled locally.
> 
> It is also possible to merge pull request branch 1 into pull request branch 2 locally in git before merging either pull request. Maybe that can minimize conflicts, but we haven't usually used that workflow.
> 
> Overall, it is helpful if contributors do their best to not edit the same sentences or paragraphs of the manuscript at the same time (in separate pull requests) because that will create more conflicts. Opening a draft pull request even if it isn't ready for final review and merging yet is one way to help notify everyone which parts are actively being edited.

I was also wondering how we could avoid conflicts in case multiple people start to work on the same section. Something like what you proposed in the last paragraph (draft pull requests) would make a lot of sense!

File: content/11.raw-data-analysis.md
Comment (agitter) [inline content/11.raw-data-analysis.md:41]: URL citations that contain special characters like `?` cause problems when getting the reference metadata.  You can fix that by wrapping everything in curly braces `{ }` like
`@{https://biognosys.com/software/spectronaut/?gclid=Cj0KCQiAoY-PBhCNARIsABcz770nuaU2SgIriS-ZJJGsC6CtzXc9AC8b9K3w5FlFDsDfGtnuUjIhankaAvegEALw_wcB}`

Note that the `url:` prefix is optional  `https:` can also be used directly when citing a URL.

Comment (dschust-r) [reply content/11.raw-data-analysis.md:41]: Thanks a lot for the feedback that's really helpful! I will fix this later today.

File: content/11.raw-data-analysis.md
Comment (agitter) [inline content/11.raw-data-analysis.md:25]: DOI citations expect just the DOI, not the doi.org DOI URL.  So like `@doi:10.1038/ncomms6277`.

The way this is cited still works because of redundancy built in, but it will be slower to build the manuscript and may be more error prone when retrieving the reference metadata.

**Pull request merged**
